ast/ddl.go

```diff
@@ -667,15 +668,16 @@ const (
 type AlterTableSpec struct {
 	node
 
-	Tp            AlterTableType
-	Name          string
-	Constraint    *Constraint
-	Options       []*TableOption
-	NewTable      *TableName
-	NewColumn     *ColumnDef
-	OldColumnName *ColumnName
-	Position      *ColumnPosition
-	LockType      LockType
+	Tp              AlterTableType
+	Name            string
+	Constraint      *Constraint
+	Options         []*TableOption
+	NewTable        *TableName
+	NewColumn       *ColumnDef
+	AppendedColumns []*ColumnDef
```

Comment:
mysql> alter table t add column c5 int after c2, add column c6 int after c3;
Query OK, 0 rows affected (0.14 sec)
Records: 0  Duplicates: 0  Warnings: 0

It can success in MySQL 5.7